sdk/voiceservices/arm-voiceservices/recordings/node/voiceservices_test/recording_communicationsgateways_create_test.json

```diff
@@ -0,0 +1,375 @@
+      },
+      "ResponseBody": {
+        "id": "/subscriptions/azure_subscription_id/providers/Microsoft.VoiceServices/locations/WESTCENTRALUS/operationStatuses/41058bb0-73a0-42f7-8e31-4d3935aca661*FC12B6141A7AD727CC4C68B1D662F210F81E881E45AB98D1A411796D57B23D5E",
+        "name": "41058bb0-73a0-42f7-8e31-4d3935aca661*FC12B6141A7AD727CC4C68B1D662F210F81E881E45AB98D1A411796D57B23D5E",
+        "resourceId": "/subscriptions/azure_subscription_id/resourceGroups/myjstest/providers/Microsoft.VoiceServices/communicationsGateways/mycg",
+        "status": "Accepted",
+        "startTime": "2023-02-22T09:36:16.1800917Z"
+      }
+    },
+    {
+      "RequestUri": "https://management.azure.com/subscriptions/azure_subscription_id/providers/Microsoft.VoiceServices/locations/WESTCENTRALUS/operationStatuses/41058bb0-73a0-42f7-8e31-4d3935aca661*FC12B6141A7AD727CC4C68B1D662F210F81E881E45AB98D1A411796D57B23D5E?api-version=2023-01-31",
+      "RequestMethod": "GET",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Accept-Encoding": "gzip,deflate",
+        "Authorization": "Sanitized",
+        "Connection": "keep-alive",
+        "Content-Type": "application/json",
+        "User-Agent": "azsdk-js-arm-voiceservices/1.0.0-beta.1 core-rest-pipeline/1.10.2 Node/v16.17.0 OS/(x64-Windows_NT-10.0.22621)",
+        "x-ms-client-request-id": "ad788274-c747-4da1-98f2-ebd9f5617908"
+      },
+      "RequestBody": null,
+      "StatusCode": 200,
+      "ResponseHeaders": {
+        "Cache-Control": "no-cache",
+        "Content-Encoding": "gzip",
+        "Content-Type": "application/json; charset=utf-8",
+        "Date": "Wed, 22 Feb 2023 09:36:26 GMT",
+        "ETag": "\u00223502fae4-0000-0100-0000-63f5e2190000\u0022",
+        "Expires": "-1",
+        "Pragma": "no-cache",
+        "Strict-Transport-Security": "max-age=31536000; includeSubDomains",
+        "Transfer-Encoding": "chunked",
+        "Vary": "Accept-Encoding",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-correlation-request-id": "6fb3d2a0-9e3a-43fb-9cd4-c5176aedec7a",
+        "x-ms-ratelimit-remaining-subscription-reads": "11994",
+        "x-ms-request-id": "410bb63e-b8eb-4de8-b18e-556990619496",
+        "x-ms-routing-request-id": "SOUTHEASTASIA:20230222T093627Z:6fb3d2a0-9e3a-43fb-9cd4-c5176aedec7a"
+      },
+      "ResponseBody": {
+        "id": "/subscriptions/azure_subscription_id/providers/Microsoft.VoiceServices/locations/WESTCENTRALUS/operationStatuses/41058bb0-73a0-42f7-8e31-4d3935aca661*FC12B6141A7AD727CC4C68B1D662F210F81E881E45AB98D1A411796D57B23D5E",
+        "name": "41058bb0-73a0-42f7-8e31-4d3935aca661*FC12B6141A7AD727CC4C68B1D662F210F81E881E45AB98D1A411796D57B23D5E",
+        "resourceId": "/subscriptions/azure_subscription_id/resourceGroups/myjstest/providers/Microsoft.VoiceServices/communicationsGateways/mycg",
+        "status": "Succeeded",
+        "startTime": "2023-02-22T09:36:16.1800917Z",
+        "properties": null
```

Comment:
@deyaaeldeen Could you take a look at the LRO failure with below message?

```
[11:12 AM] ZiWei Chen (WICRESOFT NORTH AMERICA LTD)
1) voiceservices test
[node-tests]        communicationsGateways create test:
[node-tests]      Error: Polling was unsuccessful. Expected status to have a string value or no value but it has instead: null. This doesn't necessarily indicate the operation has failed. Check your Azure subscription or resource status for more information.


```

Reply:
Patched core client has been released and this release is now unblocked.